integration_tests/mockApis/componentApi.ts

```diff
@@ -0,0 +1,44 @@
+
+const componentsNoServices: HeaderFooterMeta = {
+  activeCaseLoad: {
+    caseLoadId: 'HEI',
+    currentlyActive: true,
+    description: 'HMP Hewell',
+    type: 'INST',
+    caseloadFunction: 'GENERAL',
+  },
+  caseLoads: [
+    {
+      caseLoadId: 'HEI',
+      currentlyActive: true,
+      description: 'HMP Hewell',
+      type: 'INST',
+      caseloadFunction: 'GENERAL',
+    },
+  ],
+  services: [],
+}
+
+export const componentsNoServicesMock: { header: Component; footer: Component; meta: HeaderFooterMeta } = {
+  header: {
+    html: headerString,
+    css: [],
+    javascript: [],
+  },
+  footer: {
+    html: '<footer class="connect-dps-common-footer govuk-!-display-none-print" role="contentinfo"></footer>',
+    css: [],
+    javascript: [],
+  },
+  meta: componentsNoServices,
+}
+
+export default {
+  stubComponentsMeta: (resp = componentsNoServicesMock) => stubGet('/components.*', resp),
+}
```

Comment:
This was the tricky part - stubbing a reasonable response to Cypress, which would set the user's active caseload to HEI (Hewell) so that it matches the stub for prisoner search.